plugins/flytekit-ray/flytekitplugins/ray/models.py

```diff
@@ -1,10 +1,170 @@
+import json as _json
 import typing
 
+from flyteidl.core import tasks_pb2 as _core_task
 from flyteidl.plugins import ray_pb2 as _ray_pb2
+from google.protobuf import json_format as _json_format
+from google.protobuf import struct_pb2 as _struct
 
 from flytekit.models import common as _common
 
 
+class K8sObjectMetadata(_common.FlyteIdlEntity):
```

Comment:
I took these from `flytekit/models/task.py` instead of reusing them. I'm happy to just reuse the other models but I figured we might want to remove data loading config since that doesn't make any sense here yet.

Reply:
We're trying to get rid of these model files, so let's try to contain their existence to `flytekit/models`. In this specific case, we don't anticipate that they will diverge (due to the Lindy effect), so reusing the existing model file is the way to go.

Reply:
Updated in 185ba394f08fbfe195b2a569a5e312292f41ad3f